3 reports.sql

```diff
@@ -0,0 +1,39 @@
+--Reports:
+--1) I need to know if I have any repeat customers and what they rented
+select FullName=concat(i.CustomerFirstName,' ',i.CustomerLastName),NumberOfTimesCustomerRented= count(*)
+from Instrument i 
+group by concat(i.CustomerFirstName,' ',i.CustomerLastName)
+--if u would have customers that came more than once you would see who and what they rented
+having count(*) >1
+
+
+
+
+
+--2)I need to know the most popular category of instrument that has ever been rented
+
+
+select top 1  TotalMonthsRented=sum(i.NumberOfMonthsRented),i.Category
```

Comment:
I need to know which category is most popular (most people have rented) - not which one is rented for longest.